notebooks/problem-sets/algo_queen_2023/aq-hello-world-notepad.ipynb

```diff
@@ -44,15 +44,28 @@
    "id": "2b33bd71",
    "metadata": {},
    "source": [
-    "## Sample Problem: Applying Hadamard to 2 Qubits"
+    "## Sample Problem: Applying Hadamard to 2 Qubits\n",
+    "\n",
+    "### Instructions\n",
+    "\n",
+    "Please use this space as a notepad to test your code solutions for the Qiskit Hello World! Sample Problem. When you're tackling the Sample Problem: Applying Hadamard to 2 Qubits, and think you have the correct answer, please copy your code from there and paste it into the Sample Problem: Applying Hadamard to 2 Qubits code block and submit your answer."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "id": "5a76d8fe",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "SyntaxError",
+     "evalue": "invalid syntax (3957070354.py, line 18)",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;36m  File \u001b[0;32m\"/var/folders/mr/d9zqwqhj0ns0qy5yh2dk1qqr0000gp/T/ipykernel_9121/3957070354.py\"\u001b[0;36m, line \u001b[0;32m18\u001b[0m\n\u001b[0;31m    qc =\u001b[0m\n\u001b[0m         ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
+     ]
+    }
+   ],
```

Comment:
@quantum-kittens not sure why, but it looks like it uploaded a SyntaxError again. i will clean it up


```suggestion
   "outputs": [],
```